src/templates/blog-post.tsx

```diff
@@ -16,6 +17,8 @@ interface BlogArticleTemplateProps {
         date: string;
         title: string;
         image?: string;
+        author: string;
```

Comment:
as the author could be optional it should be typed with `?` (`author?: string`). same for the `authorSummary`

Reply:
sure, thanks, it's nice idea